[Backport release-22.05] maintainers: add Scrumplex and minion3665, tomlplusplus: init at 3.2.0, prismlauncher: init at 5.0

Minion3665 opened the pull request:
###### Description of changes

- Add myself to maintainers as in #174095 (although do not backport the package)
- Backport all commits from #193038
- Add the long key ID to the Scrumplex key, as this appears to be required in 22.05
- Init prism launcher at 5.0

This is a manual backport of #196624, as well as all its dependencies

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [x] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

Minion3665 committed:
maintainers: add minion3665

Scrumplex committed:
maintainers: add Scrumplex

Signed-off-by: Sefa Eyeoglu <contact@scrumplex.net>
Signed-off-by: Skyler Grey <skyler3665@gmail.com>

Scrumplex committed:
tomlplusplus: init at 3.2.0

Signed-off-by: Sefa Eyeoglu <contact@scrumplex.net>

Minion3665 committed:
prismlauncher: init at 5.0

Minion3665 committed:
release-notes-2205: suggest using prismlauncher

- Previously PolyMC was the suggested replacement for MultiMC
- As PolyMC is marked as insecure and prismlauncher is a replacement,
  this commit suggests using it instead

Minion3665 commented:
@ofborg eval

LunNova commented:
Result of `nixpkgs-review pr 196721` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>tomlplusplus</li>
  </ul>
</details>

Minion3665 commented:
Given that #196624 will need to be backported manually, I'd like to wait until it's merged before merging this so we can put it in the same PR

Minion3665 commented:
Result of `nixpkgs-review pr 196721` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package blacklisted:</summary>
  <ul>
    <li>nixos-install-tools</li>
  </ul>
</details>
<details>
  <summary>2 packages built:</summary>
  <ul>
    <li>prismlauncher</li>
    <li>tomlplusplus</li>
  </ul>
</details>

LunNova reviewed (approved):
Builds and runs, tested a few instances migrated from polymc and seems to be working fine.

Minion3665 committed:
multimc: suggest prismlauncher as an alternative

- Previously PolyMC was the suggested alternative
- This commit replaces that with prismlauncher, as PolyMC is no longer
  secure

SuperSandro2000 commented on maintainers/maintainer-list.nix:
```diff
@@ -11466,6 +11477,17 @@
+    github = "Scrumplex";
+    githubId = 11587657;
+    keys = [{
+      longkeyid = "rsa2048/0xC10411294912A422";
```
FYI you should regenerate a GPG key with more bits some time
